mdl/v1/transaction.go

```diff
@@ -88,3 +88,8 @@ type ActionRef struct {
 	TrxID       string `json:"trx_id"`
 	ActionIndex int    `json:"action_index"`
 }
+
+type TransactionList struct {
+	NextCursor   string                  `json:"next_cursor"`
```

Comment:
Check for consistency with `eosws`.. we use `cursor` over there I think. Make it the same.